tests/test.js

```diff
@@ -51,58 +51,88 @@ describe("Embroider build", function () {
           }
         }
       });
+
+      if (scenario === "default") {
+        it("adds integrity attribute to link elements for alternate stylesheets", function () {
+          for (const el of indexHtml.querySelectorAll("link[rel='alternate stylesheet']")) {
+            expect(el.hasAttribute("integrity")).to.be.true;
+            expect(el.getAttribute("integrity")).to.be.a("string");
+          }
+        });
+
+        it("adds integrity attribute to link elements for modules", function () {
+          for (const el of indexHtml.querySelectorAll("link[rel=module]")) {
+            expect(el.hasAttribute("integrity")).to.be.true;
+            expect(el.getAttribute("integrity")).to.be.a("string");
+          }
+        });
+
+        it("adds integrity attribute to link elements for modulepreloads", function () {
+          for (const el of indexHtml.querySelectorAll("link[rel=modulepreload]")) {
```

Comment:
For readability per above:

```suggestion
          for (const el of indexHtml.querySelectorAll('link[rel="modulepreload"]')) {
```